requirements/requirements_lightning.txt

```diff
@@ -1,6 +1,6 @@
 hydra-core>=1.2.0,<1.3
 omegaconf>=2.2,<2.3
-pytorch-lightning>=1.8.3
+pytorch-lightning>=1.8.6
```

Comment:
Hope its ok to upgrade the version here @titu1994

Reply:
Should be fine, r1.15 has been cut already so it affects only main.